### ISSUE TITLE
New audio not packaged with appimage builds

### ISSUE THREAD

**Issue by ZeeMajora** [OpenRCT2/OpenRCT2#18890]
### Operating System

Linux (OpenSUSE Tumbleweed)

### OpenRCT2 build

OpenRCT2, v0.4.3

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This issue has to do with park objects and/or asset packs

### Describe the issue

The new audio (RMC sounds, new music objects, audio replacement asset packs) is not currently packaged with appimage builds.
As a result this makes the RMC coasters silent, and results in missing object errors when trying to open parks with the new music objects enabled alongside the audio replacement asset packs not being available.

### Steps to reproduce

1. Download and run the appimage of release 0.4.3
2. Asset pack list is empty asides from the base assets
3. RMC coasters are silent
4. New music objects are not present

### Attachments

![Screenshot_select-area_20221219162040](https://user-images.githubusercontent.com/42477864/208526239-e892e4c1-d180-45fd-b63a-6bcf96cff86b.png)
![Screenshot_select-area_20221219162339](https://user-images.githubusercontent.com/42477864/208526847-c1063384-949f-4f1c-89c7-b2a617ba28b5.png)

**Comment by astraylife1** [comment]
This is happening to me with MacOS as well. I don't have any asset packs except the base graphics.

**Comment by astraylife1** [comment]
No, the latest build has not solved the issue for me on MacOS. I have the openrct2.audio.additional.parkobj but there is no asset pack for it. The additional music and asset pack do not show up. I have downloaded the additional music files and the openrct2.music.alternative.parkap file in the correct locations on my own to get them to work. Thus far I have not seen any asset packs with each build. I had to manually place those files

**Comment by janisozaur** [comment]
@ZeeMajora can you check if https://github.com/OpenRCT2/OpenRCT2/issues/18890 fixes it for you?
